starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1790,12 +1804,14 @@ sub print_report {
     if ( ( defined $self->{ra} ) and (defined $self->{dec}) and (defined $self->{roll})){
 	$o .= sprintf "RA, Dec, Roll (deg): %12.6f %12.6f %12.6f\n", $self->{ra}, $self->{dec}, $self->{roll};
     }
-    if ( defined $self->{DITHER_ON} && $self->{obsid} < $ER_MIN_OBSID ) {
-	$o .= sprintf "Dither: %-3s ",$self->{DITHER_ON};
+    if ( defined $self->{cmd_dither_status} ) {
+	$o .= sprintf "Dither: %-3s ",$self->{cmd_dither_status};
```

Comment:
This change changes the text from 'ON' to ENAB, so that's somewhat controversial.

Reply:
I think `ENAB` is more correct, but if there are downstream interface impacts (annie etc?) then it might not be worth it.